src/lib/browse/HospitalsLayer.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+  import OptionalLayer from "./OptionalLayer.svelte";
+</script>
+
+<OptionalLayer
+  name="hospitals"
+  singularNoun="hospital"
+  pluralNoun="Hospitals"
+  color="blue"
```

Comment:
Consider the colors temporary. I'll followup soon with something to let the user control how interventions are styled on the scheme page -- grouping by funding program and other fields might be useful. Not sure yet how to make contextual layers like this interact nicely with interventions.

Reply:
Out of interest why is `singluarNoun` not capitalised but `pluralNoun` is?

Reply:
One is used for "Unnamed X" and the other for the checkbox "[color] Hospitals". I'll add a comment